.gitignore

```diff
@@ -80,6 +80,7 @@ scikits
 !np_datetime_strings.c
 *.c
 *.cpp
+*.pxi
```

Comment:
Will change:

- remove `pxi` from excluding rules
- change `setup.py` to overwrite `pxi`if required comparing its timestamp.